torch/csrc/autograd/variable.h

```diff
@@ -267,6 +267,7 @@ struct Variable::Impl : public at::TensorImpl {
 
   ~Impl() override;
 
+  int64_t numel() const override;
```

Comment:
why don't you override is_contiguous, resize_dim, set_size, set_stride, set_storage_offset?  They probably aren't called yet, but it's going to be a bad experience if someone does call them...